boards/shields/eval_adxl372_ardz/eval_adxl372_ardz.overlay

```diff
@@ -12,6 +15,8 @@
 		reg = <0x0>;
 		spi-max-frequency = <DT_FREQ_M(1)>;
 		int1-gpios = <&arduino_header 13 GPIO_ACTIVE_HIGH>;
+		fifo-mode = <ADXL372_FIFO_MODE_STREAMED>;
+		fifo-watermark = <0x80>;
```

Comment:
0x80 means nothing to me as a casual reader, could use with a comment, an integer value (rather than hex perhaps), or a self readable #define or maybe some combination of all the above